k8s/pkg/registryserver/k8s_cluster_info.go

```diff
@@ -95,3 +96,38 @@ func (k *k8sClusterInfo) MonitorSubnets(empty *empty.Empty, stream registry.Clus
 		}
 	}
 }
+
+func (k *k8sClusterInfo) GetNodeIPConfiguration(ctx context.Context, nodeIPConfiguration *registry.NodeIPConfiguration) (*registry.NodeIPConfiguration, error) {
+	nodes, err := k.clientset.CoreV1().Nodes().List(metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	for i := range nodes.Items {
+		node := nodes.Items[i]
```

Comment:
Why not just `for i, node := range nodes.Items`?

Reply:
Linter does not like it because nodes.Items is array of objects, not pointers, and it make copy of an object each iteration 
```
GO111MODULE=on golangci-lint run --new-from-rev=origin/master
k8s/pkg/registryserver/k8s_cluster_info.go:106:2: rangeValCopy: each iteration copies 704 bytes (consider pointers or indexing) (gocritic)
        for _, node := range nodes.Items {

```